web/src/cypress/integration/alerts.ts

```diff
@@ -62,6 +62,42 @@ function testAlerts(screen: ScreenFormat) {
       cy.get('body').should('not.contain', 'No results') // mock alerts should show again
     })
 
+    it('should filter through alerts, active, unacknowledged, acknowledged, closed, and all', () => {
```

Comment:
This test seems a little too dependant on DB state. To be fair, the one above it is too. We should probably truncate the user favorites table before it runs to ensure a clean/known state.

But for this one testing filters, we need to create known alerts of each type to ensure the filter is working properly. Though it may or may not be worth it with the redesign work pending.

Reply:
If this is pending updates, is this something I should remove till the updates happen?